services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -22,31 +22,44 @@ class PermitAmendment(AuditMixin, Base):
     _edit_key = PERMIT_AMENDMENT_EDIT_GROUP
 
     permit_amendment_id = db.Column(db.Integer, primary_key=True)
-    mine_guid = db.Column(UUID(as_uuid=True), db.ForeignKey('mine.mine_guid'), nullable=False)
-    permit_amendment_guid = db.Column(UUID(as_uuid=True), server_default=FetchedValue())
-    permit_id = db.Column(db.Integer, db.ForeignKey('permit.permit_id'), nullable=False)
+    mine_guid = db.Column(UUID(as_uuid=True),
+                          db.ForeignKey('mine.mine_guid'),
+                          nullable=False)
+    permit_amendment_guid = db.Column(UUID(as_uuid=True),
+                                      server_default=FetchedValue())
+    permit_id = db.Column(db.Integer,
+                          db.ForeignKey('permit.permit_id'),
+                          nullable=False)
     received_date = db.Column(db.DateTime, nullable=False)
     issue_date = db.Column(db.DateTime, nullable=False)
     authorization_end_date = db.Column(db.DateTime, nullable=False)
     permit_amendment_status_code = db.Column(
-        db.String(3), db.ForeignKey('permit_amendment_status_code.permit_amendment_status_code'))
+        db.String(3),
+        db.ForeignKey(
+            'permit_amendment_status_code.permit_amendment_status_code'))
     permit_amendment_type_code = db.Column(
-        db.String(3), db.ForeignKey('permit_amendment_type_code.permit_amendment_type_code'))
+        db.String(3),
+        db.ForeignKey('permit_amendment_type_code.permit_amendment_type_code'))
     description = db.Column(db.String, nullable=True)
-    deleted_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
+    deleted_ind = db.Column(db.Boolean,
+                            nullable=False,
+                            server_default=FetchedValue())
     lead_inspector_title = db.Column(db.String, nullable=True)
     regional_office = db.Column(db.String, nullable=True)
 
     permit_amendment_status = db.relationship('PermitAmendmentStatusCode')
-    permit_amendment_status_description = association_proxy('permit_amendment_status',
-                                                            'description')
```

Comment:
What does your .vscode/settings.json file "python.formatting.yapfArgs" key look like? This looks like competing formatters.

Reply:
Apparently it didn't exist and the mds one was not taking effect, I have updated it to be the same as mds. I'll resave all these files and push up the change

```
"python.formatting.yapfArgs": [
    "--style={column_limit:100, spaces_before_comment:'15,30,50', SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN:TRUE}"
  ],
```